image_preprocessing/image_captioning_pipeline/dataloop.json

```diff
@@ -0,0 +1,394 @@
+            },
+            "condition": "{}",
+            "action": "approved"
+          }
+        ],
+        "startNodes": [
+          {
+            "nodeId": "228c8572-cd63-4d73-94bc-fe48e8e60007",
+            "type": "root",
+            "id": "e51dbf90-0fe3-45cd-a3f5-480e2930bf62"
+          }
+        ],
+        "variables": [],
+        "description": "",
+        "name": "image-captioning-template",
+        "templateKind": "org",
+        "nodes": [
+          {
+            "id": "228c8572-cd63-4d73-94bc-fe48e8e60007",
+            "inputs": [
+              {
+                "portId": "183b64c2-3ce2-4aca-bddd-05d85419a300",
+                "nodeId": "183b64c2-3ce2-4aca-bddd-05d85419a300",
+                "type": "Item",
+                "name": "item",
+                "displayName": "item",
+                "io": "input"
+              }
+            ],
+            "outputs": [
+              {
+                "portId": "00f71429-a1a7-40bb-b554-39c456878972",
+                "nodeId": "00f71429-a1a7-40bb-b554-39c456878972",
+                "type": "Item",
+                "name": "item",
+                "displayName": "item",
+                "io": "output"
+              }
+            ],
+            "name": "Dataset",
+            "type": "storage",
+            "namespace": {
+              "functionName": "clone_item",
+              "projectName": "DataloopTasks",
+              "serviceName": "pipeline-utils",
+              "moduleName": "default_module",
+              "packageName": "pipeline-utils"
+            },
+            "metadata": {
+              "position": {
+                "x": 9964.08153839183,
+                "y": 10230.9994536364,
+                "z": 0
+              },
+              "componentGroupName": "data",
+              "repeatable": true
+            }
+          },
+          {
+            "id": "019cb5b5-b7d3-42c5-afcf-d7553afea81f",
+            "inputs": [
+              {
+                "portId": "7d91d634-f3b5-4881-a055-4b8d5f43b6cc",
+                "nodeId": "7d91d634-f3b5-4881-a055-4b8d5f43b6cc",
+                "type": "Item",
+                "name": "item",
+                "displayName": "item",
+                "io": "input"
+              }
+            ],
+            "outputs": [
+              {
+                "portId": "1102c996-ab9a-49df-88ec-2fac8484ac92",
+                "nodeId": "1102c996-ab9a-49df-88ec-2fac8484ac92",
+                "type": "Item",
+                "name": "item",
+                "displayName": "item",
+                "io": "output"
+              },
+              {
+                "portId": "5e81365a-a266-46d5-8acc-d5f9ea3a7944",
+                "nodeId": "5e81365a-a266-46d5-8acc-d5f9ea3a7944",
+                "type": "Annotation[]",
+                "name": "annotations",
+                "displayName": "annotations",
+                "io": "output"
+              }
+            ],
+            "name": "Image Model 1",
```

Comment:
i would personally prefer labelling these nodes "Captioning Model" or "Image Captioning Model" for better user clarity, though it's ok also if you want to leave it.